vendor/google.golang.org/grpc/health/health.go

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-//go:generate protoc --go_out=plugins=grpc,paths=source_relative:. grpc_health_v1/health.proto
+//go:generate $GOPATH/src/istio.io/istio/bin/protoc.sh --go_out=plugins=grpc,paths=source_relative:. grpc_health_v1/health.proto
```

Comment:
@geeknoid , @kyessenov : looks like we added this manually to files right, so the change mentioned here should be done?

Reply:
looks like this is not needed.. removing vendor/ files